network/server_test.go

```diff
@@ -491,3 +499,43 @@ func TestSelfConnection_WithBootNodes(t *testing.T) {
 		})
 	}
 }
+
+func TestMinimumBootNodeCount(t *testing.T) {
+	tests := []struct {
+		name      string
+		bootNodes []string
+		isError   bool
```

Comment:
```suggestion
		shouldFail bool
```

We use the flags `shouldFail` and `shouldSucceed` in all of the other tests, so there is no reason to change it up here